models/common-images.json

```diff
@@ -4,56 +4,18 @@
   "title": "images",
   "minItems": 1,
   "items": {
-    "type": "object",
-    "properties": {
-      "@id": {
-        "$ref": "./common-string-uri-readOnly.json",
-        "description": "A unique URI that acts as globally unique ID for the image object. Based on JSON-LD node identifiers. https://www.w3.org/TR/json-ld/#node-identifiers"
-      },
-      "id": {
-        "$ref": "./common-string-uuid.json",
-        "description": "The unique id (generated by UiTdatabank) of the image."
-      },
-      "contentUrl": {
-        "$ref": "./common-string-uri-readOnly.json",
-        "description": "URL of the image file."
-      },
-      "thumbnailUrl": {
-        "$ref": "./common-string-uri-readOnly.json",
-        "description": "URL to a thumbnail image."
-      },
-      "description": {
-        "type": "string",
-        "minLength": 1,
-        "pattern": "\\S",
-        "description": "Description of the image object, in the language used in the `language` property of the image object."
-      },
-      "copyrightHolder": {
-        "$ref": "./common-copyrightHolder.json",
-        "description": "Name of the copyright holder of the image."
-      },
-      "language": {
-        "$ref": "./common-language.json",
-        "description": "Locale of text used in the image and its description."
-      }
-    },
-    "required": [
-      "id",
-      "description",
-      "copyrightHolder",
-      "language"
-    ]
+    "$ref": "./image.json"
```

Comment:
Note: I've called this `image.json` and not `common-image.json` because the `common-` are used for schemas that are usually properties. But this `image.json` can later also be used for the schema for `GET /images/{imageId}` for example. So it's more similar to `event.json`, `place.json`, ...

Reply:
Thanks for clarifying